docs/store/README.md

```diff
@@ -96,7 +96,7 @@ export class MyAppComponent {
 	counter: Observable<number>;
 
 	constructor(private store: Store<AppState>) {
-		this.counter = store.select<number>('counter');
+		this.counter = store.select<number>((state: AppState) => state.counter);
```

Comment:
Don't remove the string selector, remove the `<number>` type instead. You can't use the generic type with the string selector, which is the cause for the error.